app/services/support_interface/candidate_email_send_counts_export.rb

```diff
@@ -16,9 +17,9 @@ def data_for_export
     def emails_with_total_send_counts
       Email
         .select(
-          'mail_template, COUNT(mail_template) AS send_count, COUNT(DISTINCT "to") AS unique_recipients, MAX(created_at) AS last_sent_at',
+          'mail_template, COUNT(mail_template) AS send_count, COUNT(DISTINCT "to") AS unique_recipients, MAX(created_at) AS last_sent_at, mailer',
         )
-        .group('mail_template')
+        .group('mail_template, mailer')
```

Comment:
I used this post for reference here - https://stackoverflow.com/questions/1241352/rails-group-by-multiple-columns